romancal/pipeline/exposure_pipeline.py

```diff
@@ -71,6 +71,8 @@ def process(self, input):
 
         # make sure source_catalog returns the updated datamodel
         self.source_catalog.return_updated_model = True
+        # make sure we update source catalog coordinates afer running TweakRegStep
+        self.tweakreg.update_source_catalog_coordinates = True
```

Comment:
It looks like this attribute is set here and only here. Patching the step object like this is not a good idea.
It should be added to the step spec with a default value.

Reply:
Mairan, Nadia and I chatted briefly about this earlier today, and I had gotten confused with the related self.source_catalog.return_updated_model above.  Let's make this default False for now, but it could go into the spec I think?

Reply:
Just making sure we're on the same page, we can keep the line above in the ELP code, but we should add `update_source_catalog_coordinates` to `TweakRegStep` and set its default to `False`, correct?

Reply:
Yes, add update_source_catalog_coordinates to the tweakreg step spec with a default value of False, exactly.